var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+from spack import *
+import os
+import shutil
+
+
+class NeurodamusCore(Package):
+    """Library of channels developed by Blue Brain Project, EPFL"""
+
+    homepage = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
+    git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
```

Comment:
I would use `url` here

Reply:
Most packages use git, and that lets spack know we use a git repo by default. Is there any disadvantage?

Reply:
Since we never distribute tarballs in these projects, I would really prefer `git=` and only that. It makes obvious we only build from the repo, and `version()` will also work even if git=xxx is not provided. As discussed offline, this is now allowed since SPACK PR 3161.